support-frontend/test/selenium/subscriptions/CheckoutsSpec.scala

```diff
@@ -63,24 +64,31 @@ class CheckoutsSpec
       productPage: ProductPage,
       paymentFunction: CheckoutPage => Unit,
   ): Unit = {
-    val testUserRequest = new IdapiTestUserRequest()
-    testUserRequest.getCookies() match {
-      case Left(error) => fail(error)
-      case Right(cookies) => new PostDeployTestUser(driverConfig, Some(cookies))
-    }
+    new PostDeployTestUser(driverConfig)
+
+    Given("that a user is signed in")
+    val signInPage = new SignInPage
+    goTo(signInPage)
+    signInPage.signIn()
 
     Given("that a user goes to the UK product page")
     goTo(productPage)
 
+    checkoutPage.addTestUserCookies("SupportPostDeployTestF")
+
     Given(s"that a user goes to the $checkoutName checkout page")
-    goTo(checkoutPage)
+    goTo(
+      checkoutPage,
+    ) // TODO: why is this recognised as a test user on https://support.theguardian.com/uk/contribute but not here?
 
     Then(s"they should be redirected to the $checkoutName checkout page")
     assert(checkoutPage.pageHasLoaded)
+    checkoutPage.disableConsentBanner()
 
     Given("the user fills in their details correctly")
     checkoutPage.fillForm()
 
+    Thread.sleep(1000)
```

Comment:
Maybe using `WebDriverWait` with a timeout would work, like we do [in waitUntil](https://github.com/guardian/support-frontend/blob/d567fbd8c9f5c0726e13634253db19d319a75371/support-frontend/test/selenium/util/Browser.scala#L116)?

Reply:
There seemed to be some precedents for doing it this way. eg. https://github.com/guardian/support-frontend/blob/eb4fa9e84801fdd8ddbf69a6adc93d9125aa8487/support-frontend/test/selenium/contributions/pages/ContributionsLanding.scala#L154
but I agree that it's not clear what is being waited for.  It needed a delay of some kind to make the tests run more reliably.  I'll see what I can do.

Reply:
Grand, see what you can do – I’m hoping we can change a bunch of the sleeps to retries, but I haven’t looked into it yet (and that wouldn’t be part of this PR). Would be nice to avoid adding more if we can, but it’s not urgent.